lab_1_classify_by_unigrams/main.py

```diff
@@ -1,68 +1,104 @@
 """
 Lab 1
-Language detection
+Language detection turi ip ip
 """
 
 
 def tokenize(text: str) -> list[str] | None:
-    """
-    Splits a text into tokens, converts the tokens into lowercase,
-    removes punctuation, digits and other symbols
-    :param text: a text
-    :return: a list of lower-cased tokens without punctuation
-    """
-
+    if type(text) is str:
+        tokenized = []
+        for token in text:
+            if token.isalnum() and not token.isnumeric():
+                tokenized.append(token.lower())
+        return tokenized
+    else:
+        return None
 
 def calculate_frequencies(tokens: list[str] | None) -> dict[str, float] | None:
-    """
-    Calculates frequencies of given tokens
-    :param tokens: a list of tokens
-    :return: a dictionary with frequencies
-    """
-
+    if type(tokens) is list:
+        dict_instances = {}
```

Comment:
tooooooo much code for this small task - please, simplify

Reply:
done